code/game/objects/items/devices/radio/radio.dm

```diff
@@ -407,7 +407,7 @@ GLOBAL_LIST_INIT(default_medbay_channels, list(
 		jobname = "Cyborg"
 
 	// --- Personal AI (pAI) ---
-	else if(istype(M, /mob/living/silicon/pai))
+	else if(ispAI(M))
```

Comment:
somewhat related, but probably out of scope. This `ispAI` is really close to `isAI`, which might lead to confusion in the future. Maybe something like `ispersonalAI` is better